upload/system/library/log.php

```diff
@@ -30,11 +30,11 @@ public function __construct(string $filename) {
 	/**
 	 * Write
 	 *
-	 * @param string $message
+	 * @param mixed $message
```

Comment:
A more accurate type:
```suggestion
	 * @param string|array<string> $message
```